### ISSUE TITLE
Update line heights

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#1194]
According to the [WCAG 2.1 on text spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html), line-heights should be 1.5 for body text.

Design specifications: https://www.figma.com/file/ojCcgC5Zd12eUSzq6V5m24/Foundations?node-id=3%3A950&t=uemycRw9g7CNz7E7-0

```[tasklist]
### Tasks
- [x] Default line-height should be 1.5
- [x] Header line-height should be 1.2
- [x] Deprecate old line-height variables
- [x] Update storybook foundation/typography documentation
```
